packages/fxa-content-server/app/scripts/views/confirm_signup_code.js

```diff
@@ -26,9 +27,12 @@ class ConfirmSignupCodeView extends FormView {
     return proto.afterVisible
       .call(this)
       .then(() => this.broker.persistVerificationData(account))
-      .then(() =>
-        this.invokeBrokerMethod('beforeSignUpConfirmationPoll', account)
-      );
+      .then(() => {
+        this.waitForSessionVerification(this.getAccount(), () => {
```

Comment:
Same comment from other PR, it isn't clear that the `waitForSessionVerification` handles email bounces.